libs/config_engine.py

```diff
@@ -152,10 +160,34 @@ def get_video_sources(self):
                     else:
                         src['should_send_email_notifications'] = False
                         src['should_send_slack_notifications'] = False
-                    src['daily_report'] = self.config.getboolean(title, 'DailyReport')
-                    src['daily_report_time'] = section.get('DailyReportTime') or '06:00'
                     sources.append(src)
             return sources
         except:
             # Sources are invalid in config file. What should we do?
-            raise RuntimeError("Sources invalid in config file")
+            raise RuntimeError("Invalid sources in config file")
+
+    def get_areas(self):
+        try:
+            areas = []
+            for title, section in self.config.items():
+                if title.startswith('Area_'):
+                    area = self.get_entity_with_notifications(title, section)
+                    area['type'] = 'Area'
+                    area['occupancy_threshold'] = int(section['OccupancyThreshold'])
+                    if 'Cameras' in section and section['Cameras'].strip() != "":
+                        area['cameras'] = section['Cameras'].split(',')
+
+                    if area['notify_every_minutes'] > 0 and (area['violation_threshold'] > 0 or area['occupancy_threshold'] > 0):
+                        area['should_send_email_notifications'] = 'emails' in area
+                        app_config = self.config["App"]
+                        area['should_send_slack_notifications'] = bool(bool(app_config["SlackChannel"]) and
+                                                                      strtobool(
```

Comment:
Can't you use the get_boolean function instead of strtobool? To be consistent with handling bolleans accross the app